cmd/register/register_util.go

```diff
@@ -215,15 +214,15 @@ func hydrateIdentifier(identifier *core.Identifier) {
 		identifier.Domain = config.GetConfig().Domain
 	}
 	if identifier.Version == "" || identifier.Version == registrationVersionPattern {
-		identifier.Version = rconfig.DefaultFilesConfig.Version
+		identifier.Version = defaultFilesConfig.Version
 	}
 }
 
 func hydrateTaskSpec(task *admin.TaskSpec, sourceCode string) error {
 	if task.Template.GetContainer() != nil {
 		for k := range task.Template.GetContainer().Args {
 			if task.Template.GetContainer().Args[k] == "" || task.Template.GetContainer().Args[k] == registrationRemotePackagePattern {
-				remotePath, err := getRemoteStoragePath(context.Background(), Client, rconfig.DefaultFilesConfig.SourceUploadPath, sourceCode, rconfig.DefaultFilesConfig.Version)
+				remotePath, err := getRemoteStoragePath(context.Background(), Client, defaultFilesConfig.SourceUploadPath, sourceCode, defaultFilesConfig.Version)
```

Comment:
I think the main agenda was to not use such config directly in these internal functions and be used only the first level command function .
Rest of the places we want to pass the config as parameter wherever it's used

Reply:
@pmahindrakar-oss thanks for your review
I've been pushed a new commit
Could you please review it?